web_external/ImagesGallery/Detail/imageDetailsPage.jade

```diff
@@ -86,9 +86,10 @@
       tr
         td.isic-image-details-table-section-header(colspan=2) Tags
       tr
-        td
+        td(colspan=2)
           each value in image.get('notes').tags
-            span.label.label-default= value
-        td
+            //- Labels separated by a space
+            span.label.label-default.isic-image-details-tag= value
+            | &#x20;
```

Comment:
This obviously isn't a big deal, but IMO, the `|` often makes rendering harder to control. For mixed variable and literal content, I'd just interpolate the variables into a single line, as:
```
span... #{value} &#x20;
```